oviewer/document.go

```diff
@@ -59,6 +59,8 @@ type Document struct {
 	// alignConv is an interface that converts alignment.
 	alignConv *align
 
+	cond *sync.Cond
```

Comment:
Do you really need this?

I have never faced sync.Cond until now, so I looked at documentation

https://pkg.go.dev/sync#Cond
> For many simple use cases, users will be better off using channels than a Cond

Reply:
> Do you really need this?
> 
> I have never faced sync.Cond until now, so I looked at documentation
> 
> https://pkg.go.dev/sync#Cond
> 
> > For many simple use cases, users will be better off using channels than a Cond

That's a good point.
I thought it would be best to use Broadcast (), which does not have to be received or does not need to receive more than one.
I think it is difficult to implement it without mistakes on a channel.